RPIPs/RPIP-42.md

```diff
@@ -23,18 +23,22 @@ This work is based on prior work; a copy can be found [here](../assets/rpip-42/b
 
 ## Specification
 - The protocol SHALL be able to penalize stake at the node level
-- Node Operators with <`base_mp_num` base minipools SHALL be able to make a base minpool
-  - A base minipool SHALL have a `base_mp_bond` bond from the Node Operator
-- Node Operators with >=`base_mp_num` base minipools SHALL be able to make a satellite minipool
-  - A satellite minipool SHALL have a `satellite_mp_bond` bond from the Node Operator
-- If a Node has satellite minipools active while that Node has <`base_mp_num` base minipools active, funds SHALL NOT be withdrawable by the Node Operator
-  - It is NOT RECOMMENDED to exit base minipools unless all satellite minipools have been exited
-  - A user MAY exit satellite minipools to allow for fund withdrawal 
-  - The protocol MAY or MAY NOT provide a path to upgrade a satellite minipool to a base minipool (to allow for fund withdrawal)
+- Minipool deposits SHALL be disabled
```

Comment:
This confused me at first read (thought it was an incomplete reference to a node being penalized.)
Could make it more explicit this is about old minipools (non-megapool validators.) 
(Why is it necessary to disable these immediately?)

Reply:
I'll say "legacy minipool" and make it a softer SHOULD. I think they will be categorically worse and see no reason to give users an extra way to shoot themselves in the foot.